jobs/list.ts

```diff
@@ -34,6 +35,7 @@ export const allJobs: CSCronJob[] = [
     // every 10 minutes during the day (to distribute load and send out notifications faster)
     { cronTime: '00 */10 * * * *', jobFunction: Notification.checkReminders },
     // each night - database cleanups
+    { cronTime: '00 00 05 * * *', jobFunction: anonymiseAttendanceLog },
```

Comment:
If you alao add this to manualExecution.ts, you can run it via GraphQL with _executeJob, thats easier to test ...